pkg/metronome/client.go

```diff
@@ -76,3 +78,26 @@ func (c *Client) Jobs(opts ...JobsOption) ([]Job, error) {
 
 	return jobs, err
 }
+
+// RunJob triggers a run of the job with a given runID right now.
+func (c *Client) RunJob(runID string) error {
```

Comment:
The APi says it returns a response body : 
```
{
  "completedAt": null,
  "createdAt": "2016-07-15T13:02:59.735+0000",
  "id": "20160715130259A34HX",
  "jobId": "prod",
  "status": "STARTING",
  "tasks": []
}
```
We should also return it from this method.